installers/linux/alpine/tar/build.gradle

```diff
@@ -120,7 +120,7 @@ task packageDebugSymbols(type: Exec) {
     outputs.file tarFile
     workingDir buildDir
     commandLine 'bash', '-c',
-        "tar -czf ${tarFile} ${binaryResultName}/**/*.diz"
+        "tar -czf ${tarFile} \$(find ${binaryResultName} -name \"*.diz\")"
```

Comment:
It may be better to use a gradle tar task with an include filter, but not a big deal.